test/features/environment.py

```diff
@@ -9,7 +9,7 @@
     '../../'))
 
 # Base URL for relative paths resolution.
-BASE_URL = 'http://ckan:5000/'
+BASE_URL = 'http://ckan-qld.local:5000/'
```

Comment:
Why change this?

Reply:
\+

Reply:
This will be fixed and pushed by Alex shortly.